docs/tutorials/style_transfer/demo_msg_net.py

```diff
@@ -0,0 +1,67 @@
+"""1.Getting Started with Pre-trained MSG-Net
+==========================================
+
+.. image:: https://github.com/zhanghang1989/MSG-Net/blob/master/images/figure1.jpg?raw=True
+    :width: 55%
+    :align: left
+
+This tutorial shows how to use pre-trained MSG-Net for real-time style transfer. The network
+was pre-trained using MS-COCO dataset and 21 painting style images.
+
+.. note::
+    Hang Zhang, and Kristin Dana. "Multi-style Generative Network for Real-time Transfer."::
+
+        @article{zhang2017multistyle,
+            title={Multi-style Generative Network for Real-time Transfer},
+            author={Zhang, Hang and Dana, Kristin},
+            journal={arXiv preprint arXiv:1703.06953},
+            year={2017}
+        }
+"""
+import mxnet as mx
+import gluoncv
+##############################################################################
+# Get the content and style images
+# --------------------------------
+#
+# download the example images
+content_url = 'https://github.com/dmlc/web-data/blob/master/mxnet/example/' + \
+    'style_transfer/images/content/venice-boat.jpg?raw=True'
+style_url = 'https://github.com/dmlc/web-data/blob/master/mxnet/example/' + \
+    'style_transfer/images/styles/candy.jpg?raw=True'
+content_fn = 'venice-boat.jpg'
+style_fn = 'candy.jpg'
+gluoncv.utils.download(content_url, content_fn)
```

Comment:
Please explicitly call `path_name` as in https://github.com/dmlc/gluon-cv/pull/133/files .

Or alternatively, use the url with prefix `raw.githubusercontent.com`.

Reply:
Do you suggest using `path=content_fn ` instead of positional params `content_fn`?